tools/lsvast/CMakeLists.txt

```diff
@@ -9,3 +9,19 @@ endif ()
 add_executable(lsvast lsvast.cpp)
 target_link_libraries(lsvast PRIVATE vast::libvast vast::internal)
 install(TARGETS lsvast DESTINATION "${CMAKE_INSTALL_BINDIR}")
+
+find_program(ASCIIDOCTOR asciidoctor)
```

Comment:
The input looks very simple, can't we use pandoc instead and make adjustments to the source if necessary?

Reply:
pandoc doesn't render as nicely, I tried this first. There's a massive difference in quality. I would rather do it the other way around and switch to using asciidoctor for vast.1 as well.

Reply:
Then let's not add a new documentation format just like that and rewrite this page in Markdown.
Switching to adoc should be discussed properly first.

Reply:
Please check back.